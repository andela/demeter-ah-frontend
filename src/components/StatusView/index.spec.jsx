import React from 'react';
import { shallow } from 'enzyme';
import Status from './index';

describe('StatusView Component', () => {
  it('Should render without errors', () => {
    const component = shallow(<Status output="welcome" />);
    expect(component).toMatchSnapshot();
  });
});
