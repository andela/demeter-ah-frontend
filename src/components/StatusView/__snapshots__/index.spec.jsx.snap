// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusView Component Should render without errors 1`] = `
<div
  className="w-full h-full container"
>
  <h3
    className="status"
  />
  <div
    className="actions"
  />
</div>
`;
